ProteGO/Common/DangerStatusManager/DangerStatusManagerSpecs.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import Quick
+import Nimble
+import Valet
+@testable import ProteGO
+
+//swiftlint:disable force_unwrapping
+class DangerStatusManagerSpecs: QuickSpec {
+    override func spec() {
+        describe("DangerStatusManager") {
+            var sut: DangerStatusManager!
+            var testValet: Valet!
+            var gcpClientMock: GcpClientMock!
+
+            beforeEach {
+                testValet = Valet.valet(with: Identifier(nonEmpty: "SecretsGeneratorSpecs")!, accessibility: .always)
```

Comment:
Czy użycie `Valet` nie powinno być ukryte przez jakąś cienką abstrakcję? aktualny stan tak naprawdę zapisuje wartości tworząc z tego nie test jednostkowy a integracyjny.

Reply:
Dobra uwaga, wprowadziłem warstwę KaychainProvidera ponad uzycie Valeta. W testach jest to zwykły słownik